js/languages/en_US.json

```diff
@@ -890,12 +894,16 @@
   "wallet": {
     "title": "Wallet",
     "closeLink": "Close wallet",
+    "zeroSupportedCurs": "None of the wallet currencies are currently supported by this client.",
```

Comment:
This was a little confusing when I tested it and saw it in the wallet. Maybe "None of the wallets provided by your server use currencies supported by this client.", so it's more clear the issue is a server/client mismatch?